core/src/main/java/nl/nn/adapterframework/soap/SoapWrapperPipe.java

```diff
@@ -332,17 +340,18 @@ public void setServiceNamespace(String string) {
 	}
 
 	/** (only used when direction=<code>wrap</code>) Stylesheet to create the content of the soap header. As input for this stylesheet a dummy xml string is used. Note: outputType=<code>xml</code> and xsltVersion= */
-	public void setSoapHeaderStyleSheet(String string){
+	public void setSoapHeaderStyleSheet(String string) {
 		this.soapHeaderStyleSheet = string;
 	}
 
 	/** (only used when direction=<code>wrap</code>) Stylesheet to apply to the input message. Note: outputType=<code>xml</code> and xsltVersion=2 */
-	public void setSoapBodyStyleSheet(String string){
+	public void setSoapBodyStyleSheet(String string) {
 		this.soapBodyStyleSheet = string;
 	}
 
 	/**
 	 * (only used when direction=<code>unwrap</code>) If <code>true</code>, namespaces (and prefixes) in the content of the soap body are removed
+	 *
```

Comment:
Dit hoeft toch niet?

Reply:
Je bedoelt die lege Javadoc regels, die de formatter heeft toegevoegd?

Reply:
Yes. het voelt een beetje onnodig, het voegt niets toe, behavle dat je meer moet scrollen 😄

Reply:
Ja, dat sluit dan wel weer beter aan bij de millenials ;-) Maar die werken dan weer niet in het core team :-P